src/atoms/Button/Button.stories.js

```diff
@@ -1,11 +1,116 @@
 import React from 'react';
 import { storiesOf } from '@storybook/react';
 import { action } from '@storybook/addon-actions';
+import { View, ScrollView } from 'react-native';
+import styled from 'styled-components/native';
+
 import Button from './Button';
 
+const StyledContainer = styled.View`
+  flex: 1;
+  justify-content: space-between;
+`;
+
 storiesOf('Button', module)
   .addParameters({
```

Comment:
Knobs please!

Reply:
I am currently thinking of the best way to show the variations.